### ISSUE TITLE
URL deprecation in source

### ISSUE THREAD

**Issue by GabrieleLabanca** [schireson/pytest-mock-resources#120]
When converting a Postgres fixture to url as in [the docs](https://pytest-mock-resources.readthedocs.io/en/latest/database.html#custom-engines), I get the following error
```
tests/test_sample_db_mocking.py::test_sql_sum
  /home/gab/doc/supplybrain-data-validator/.venv/lib/python3.8/site-packages/pytest_mock_resources/fixture/database/generic.py:34: SADeprecationWarning: Calling URL() directly is deprecated and will be disabled in a future release.  The public constructor for URL is now the URL.create() method.
    return URL(
```
Which relates to [this line](https://github.com/schireson/pytest-mock-resources/blob/7fa4fda9fd943aad01a53f7c5355b56fcc3da290/src/pytest_mock_resources/fixture/database/generic.py#L34).

I expect that the correction would be simply something like 
```
return URL.create(
```

**Comment by DanCardin** [comment]
Ah yes. This is mostly complicated by the fact that it deprecated the <1.4 syntax in the same version that it added the new method, so there's not a fantastic way to support both.

Not that it's impossible/hard, just not as straightforward as not using the deprecated constructor!

**Comment by DanCardin** [comment]
Should be fixed in 2.1.3

**Comment by GabrieleLabanca** [comment]
Great, thanks for the answer :)
